behaviour_and_protocol/consulting_101.md

```diff
@@ -1,26 +1,81 @@
-# Consulting 101
-At Abtion, we work with our clients to create world-class solutions that provide value to their businesses. As a consulting company, we are fortunate to have collaborated with a wide range of companies over the years from solo entrepreneurs to large Danish corporations. Key to this success is that we acknowledge that every client is different and has different needs.
-
-If you are coming from a product company, it can be unclear what consulting is, and what your new role as a consultant requires. Our goal with this document is to provide a clearer overview of consulting, dos and don'ts, etiquette, etc. We believe that training people and bringing them closer to the client ensures that we can give clients better advice and products. As a reminder, please note that the following are guidelines and not strict rules.
-
-## Client communication
-- Recognize the client's style of communication. You should not aim to mirror it but it gives you a sense of the tone that they might expect from you.
-- We use a digital project management tool (Asana is our default one) as our collaboration platform for prioritizing work with the client. We prefer our clients to be engaged in this collaboration but it is ultimately your responsibility to add and prioritize tasks. Asana is primarily available for the client to provide transparency and as a communication channel. As Asana is public to the client, we expect you to communicate professionally in comments, task descriptions, etc.
-- We want everyone independent of their role to participate actively in client communication both during meetings but also in Slack, Asana, e-mails, or wherever the communication happens. Remember to chat with your team before you reach out to the client. Your colleague might already have asked the same question, and we want to avoid spamming the client with questions. For a more in-depth introduction to how we facilitate team meetings at Abtion, please have a look at [this document](./how_to_facilitate_team_meetings.md).
```

Comment:
Previously, we had a reference to ./how_to_facilitate_team_meetings.md
Might still be relevant to link to it somewhere.

Reply:
Not merge blocking, the PR looks good.
Just something to consider.